src/0-routes/demo/table.tsx

```
@@ -49,13 +50,16 @@ const fuzzyFilter: FilterFn<any> = (row, columnId, value, addMeta) => {
 };
 
 // Define a custom fuzzy sort function that will sort by rank if the row has ranking information
+// biome-ignore lint/suspicious/noExplicitAny: demo code
 const fuzzySort: SortingFn<any> = (rowA, rowB, columnId) => {
 	let dir = 0;
 
 	// Only sort by rank if the column has ranking information
 	if (rowA.columnFiltersMeta[columnId]) {
 		dir = compareItems(
+			// biome-ignore lint/style/noNonNullAssertion: demo code
 			rowA.columnFiltersMeta[columnId]?.itemRank!,
```

Biome (2.3.14)

[error] Forbidden non-null assertion after optional chaining.

Optional chaining already handles nullish values. Using non-null assertion defeats its purpose and may cause runtime errors.
Consider using the nullish coalescing operator `??` or optional chaining throughout the chain instead.

(lint/suspicious/noNonNullAssertedOptionalChain)